criu/img-remote.c

```diff
@@ -178,6 +179,56 @@ static int setup_UNIX_client_socket(char *path)
 	return sockfd;
 }
 
+/*
+ * Using a pipe for image file transfers instead of the UNIX socket can
+ * greatly improve performance. Data can be further spliced() by the proxy
+ * to transfer data with minimal overhead.
+ * Transfer rates of up to 15GB/s can be seen with this technique.
+ */
+#define SEND_FD_FOR_READ	0
+#define SEND_FD_FOR_WRITE	1
+#define RECV_FD			2
+static int upgrade_to_pipe(int sockfd, int mode)
+{
+	return sockfd;
```

Comment:
Is this `return sockfd;` intentional?

Reply:
Good catch, wow.

Not sure how this slipped through the cracks. Sorry.